code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -28,6 +28,9 @@
 				for(var/datum/reagent/A in reagents.reagent_list)
 					R += A.id + " ("
 					R += num2text(A.volume) + "),"
+			if(isturf(M) && reagents.has_reagent("plasma"))
+				log_game("[thrownby]([thrownby.ckey]) splashed plasma at [target.x] [target.y] [target.z].")
+				message_admins("[thrownby]([thrownby.ckey]) splashed plasma at [target.x] [target.y] [target.z].")

```

Comment:
Yes, also use `[COORD(target)]` for log_game and `[ADMIN_COORDJMP(target)]` for message_admins. 

I'm pretty sure we can make it for all thrown beakers/reagent containers instead of only when it contains plasma(or move this to thrower's attack_log).